### DISCUSSION TITLE
Would it be possible to get a new release cut?

### DISCUSSION THREAD

**Discussion by ckipp01** [foundweekends/giter8#705] [Q&A]
👋🏼 hello! I recently made a [Mill plugin](https://github.com/ckipp01/mill-giter8) for Giter8, which was a ton of fun! When I pushed everything up to GitHub for the first time and submitted my deps to the GitHub Dependency Submission API I got a notification right away about a security vulnerability since the latest version of the giter8-lib brings it in.

```
├─ org.foundweekends.giter8:giter8-lib_2.13:0.14.0
│  ├─ commons-io:commons-io:2.11.0
│  ├─ org.antlr:ST4:4.3.1
│  │  └─ org.antlr:antlr-runtime:3.5.2
│  ├─ org.codehaus.plexus:plexus-archiver:2.7.1
│  │  ├─ org.codehaus.plexus:plexus-container-default:1.0-alpha-9-stable-1
│  │  │  └─ org.codehaus.plexus:plexus-utils:1.0.4 -> 3.0.8 (possible incompatibility)
│  │  ├─ org.codehaus.plexus:plexus-io:2.2
│  │  │  └─ org.codehaus.plexus:plexus-utils:3.0.7 -> 3.0.8
│  │  └─ org.codehaus.plexus:plexus-utils:3.0.8
│  ├─ org.eclipse.jgit:org.eclipse.jgit.pgm:5.13.0.202109080827-r
│  │  ├─ args4j:args4j:2.33
│  │  ├─ log4j:log4j:1.2.15 -> 1.2.17
```

Would it be possible to get a release cut that includes https://github.com/foundweekends/giter8/commit/55b649b5e2366c07b58fc0f6d1d803a691c6b6f0 which would avoid having the security alarms going off?

**Comment by xuwei-k** [comment]
> release cut that includes https://github.com/foundweekends/giter8/commit/55b649b5e2366c07b58fc0f6d1d803a691c6b6f0



<img width="1318" height="454" alt="image" src="https://github.com/user-attachments/assets/eb8e7f9f-a3bf-4910-9dd9-a107da78faca" />
